test/e2e/docker/Dockerfile

```diff
@@ -3,8 +3,8 @@
 # instead of spending time compiling them.
 FROM golang:1.15
 
-RUN apt-get update -y && apt-get upgrade -y
-RUN apt-get install -y libleveldb-dev librocksdb-dev
+RUN apt-get -qq update -y && apt-get -qq upgrade -y >/dev/null
```

Comment:
why do u need to suppress the output?

Reply:
apt-get outputs a ton of useless noise which makes the CI logs harder to read.